workflow-report/action.yml

```diff
@@ -0,0 +1,76 @@
+name: 'Report failed job to VK Team chat'
+description: 'This action creates a message about failed job and sends it to VK Teams chat'
```

Comment:
`This action composes a message about the failed job and sends it to VK Teams chat`. Please also add this description to README.md.

Reply:
done